### ISSUE TITLE
Open ephys to intan RHA

### ISSUE THREAD

**Issue by anacastropsico** [open-ephys/plugin-GUI#606]
Hello, I have itan's oldest registration system, RHA, I would like to know if it is possible to use open ephys with it, as I saw a possibility only for RHD.

**Comment by jsiegle** [comment]
Is this the system you'd like to use?

![image](https://github.com/open-ephys/plugin-GUI/assets/200366/aba4aeb2-50d9-488e-b1e3-f66ac91c5b1e)

This was supported in the earliest versions of Open Ephys software, but we no longer maintain a plugin for it.

**Comment by anacastropsico** [comment]
Yes this is. But where I can to access this versions? Is similar to Iran demo?

**Comment by jsiegle** [comment]
Here's the source code for that version: https://github.com/open-ephys/gui

You'll need to compile it yourself, and you'll definitely encounter some bugs. If you already have the Intan demo software working, I'd recommend using that.

**Comment by anacastropsico** [comment]
You believe be possible use this version you send me?

**Comment by jsiegle** [comment]
Is there anyone in your group who knows C++ and could try to compile the software? This version should work to acquire from the RHA device, but unfortunately we can't support it because it's 8 years old at this point.

**Comment by anacastropsico** [comment]
yes I'm trying with friends who are c++ programmers.
Thank you for your response and attention
